lib/Client.js

```diff
@@ -554,7 +539,8 @@ class Client extends EventEmitter {
     * @arg {String} [command.description] The command description (Slash Commands Only)
     * @arg {Array<Object>} [command.options] An array of [command options](https://discord.com/developers/docs/interactions/application-commands#application-command-object-application-command-option-structure)
     * @arg {Number} [type=1] The type of application command, 1 for slash command, 2 for user, and 3 for message
-    * @arg {Boolean} [command.defaultPermission=true] Whether the command is enabled by default when the app is added to a guild
+    * @arg {String} [command.defaultMemberPermissions] The default member permissions represented as a
+    * @arg {Boolean} [command.DMPermission] The default role permissions
```

Comment:
```suggestion
    * @arg {Boolean} [command.dmPermission] The default role permissions
```
keys should be camel cased